packages/core/src/codewhisperer/commands/startSecurityScan.ts

```diff
@@ -255,7 +255,7 @@ export async function startSecurityScan(
         codeScanState.setToNotStarted()
         codeScanTelemetryEntry.duration = performance.now() - codeScanStartTime
         codeScanTelemetryEntry.codeScanServiceInvocationsDuration = performance.now() - serviceInvocationStartTime
-        await emitCodeScanTelemetry(codeScanTelemetryEntry)
+        await emitCodeScanTelemetry(codeScanTelemetryEntry, scope)
```

Comment:
Does this prevent gitDirSize invocation when it comes to auto security scan? Then how about manual project level security scan?

Reply:
Manual scan is triggered once so this should be executed only once, which is exact functionality before 4/30